README.md

```diff
@@ -30,48 +30,94 @@ java -jar app-v-0.1.0.war
 ```
 
 
-<!--
+
  ## Running in Docker
 
 To run the application using Docker you have to install Docker and docker-compose on your local computer.  To install Docker see [here](https://docs.docker.com/get-docker/) and to install docker-compose see [here](https://docs.docker.com/compose/install/). 
 
 Assuming you have installed both Docker and docker-compose, to run the Onto Viewer application together with its front-end on your local machine, please make sure that you have the following folder setup:
 
 ```
-root-dir/
-    onto-viewer/                 (https://github.com/edmcouncil/onto-viewer)
-        docker/
-            runtime/
-                server/
-                    config/      <- config files go here
-                        ...
-                    ontologies/  <- ontologies go here
-                        ...
-        docker-compose.yaml
-        ...
-    html-pages/                  (https://github.com/edmcouncil/html-pages)
-        ...
++---onto-viewer
+|   |   .gitignore
```

Comment:
I am not sure that I got it right, but shouldn't the read:
`| .gitignore`
A similar for other files in the root folder of onto-viewer.